tests/test_utils_misc/test_return_with_argument_inside_generator.py

```diff
@@ -12,7 +15,7 @@ def _indentation_error(*args, **kwargs):
 
 def top_level_return_something():
     """
-docstring
+    docstring
```

Comment:
We need to disable this in some way, because these tests are for testing such code.